commands/raid_from_webhook.php

```diff
@@ -176,8 +176,31 @@ function isPointInsidePolygon($point, $polygon) {
     }
 
     $form = 0;
-    if ( isset($raid['message']['form']) ) {
+    if ( isset($raid['message']['form']) && $raid['message']['form'] != "0") {
+        // Use the form provided in webhook if it's valid
         $form = $raid['message']['form'];
+    }elseif($pokemon != 0) {
+        // Else look up the normal form's id from pokemon table unless it's an egg
+        try {
+            $query = "
+                SELECT pokemon_form_id FROM pokemon
+                WHERE
+                    pokedex_id = :pokemon AND
+                    pokemon_form_name = 'normal'
+                LIMIT 1
```

Comment:
Why limit 1? 🤔 
Can it happen that we get more than 1 result and therefore you limit it to 1?

Reply:
Should not be possible if the user runs our standard .sql. But ye, that is actually pretty useless, since fetch only gets the first row anyway.

Reply:
It just occured to me that having limit could have some performance benefits. We probably should add more of those (in update statements) in raid_from_webhook to benefit users with larger mapping systems.

Reply:
Better performance is always good 😁